tasks/generate_certificate.yaml

```diff
@@ -0,0 +1,48 @@
+---
+
+# TODO: Figure out how to use openssl from ansible
+#- name: Generate a Self Signed OpenSSL certificate
+#  openssl_certificate:
+#    path: /tmp/{{ broker_name }}-cert/cert.pem
+#    privatekey_path: /tmp/{{ broker_name }}-cert/key.pem
+#    subject: "/CN={{ broker_name }}.{{ broker_name }}.svc"
+#    provider: selfsigned
+- name: 'Create OpenSSL Cert for Broker'
```

Comment:
just FYI I don't think the task name is displayed in the output for blocks.

Reply:
That's what I get for trusting the example :sunglasses: http://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html#id1